src/PDS4_NH_IngestLDD.xml

```diff
@@ -49,20 +86,85 @@
+    <DD_Value_Domain>
+      <enumeration_flag>false</enumeration_flag>
+      <value_data_type>ASCII_Real</value_data_type>
+    </DD_Value_Domain>
+  </DD_Attribute>
+  
+  <DD_Attribute>
+    <name>coefficient_1</name>
+    <version_id>1.0</version_id>
+    <local_identifier>coefficient_1</local_identifier>
+    <nillable_flag>false</nillable_flag>
+    <submitter_name>MKC</submitter_name>
+    <definition>INADEQUATE Gives the value of the 1st term coeffcient.</definition>
+    <DD_Value_Domain>
+      <enumeration_flag>false</enumeration_flag>
+      <value_data_type>ASCII_Real</value_data_type>
+    </DD_Value_Domain>
+  </DD_Attribute>
+  
+  <DD_Attribute>
+    <name>coefficient_2</name>
+    <version_id>1.0</version_id>
+    <local_identifier>coefficient_2</local_identifier>
+    <nillable_flag>false</nillable_flag>
+    <submitter_name>MKC</submitter_name>
+    <definition>INADEQUATE Gives the value of the 2nd term coeffcient.</definition>
+    <DD_Value_Domain>
+      <enumeration_flag>false</enumeration_flag>
+      <value_data_type>ASCII_Real</value_data_type>
+    </DD_Value_Domain>
+  </DD_Attribute>
+  
+  <DD_Attribute>
+    <name>coefficient_3</name>
     <version_id>1.0</version_id>
+    <local_identifier>coefficient_3</local_identifier>
+    <nillable_flag>false</nillable_flag>
+    <submitter_name>MKC</submitter_name>
+    <definition>INADEQUATE Gives the value of the 3rd term coeffcient.</definition>
+    <DD_Value_Domain>
+      <enumeration_flag>false</enumeration_flag>
+      <value_data_type>ASCII_Real</value_data_type>
+    </DD_Value_Domain>
+  </DD_Attribute>
+  
+  <DD_Attribute>
+    <name>detector_name</name>
+    <version_id>1.1</version_id>
     <local_identifier>detector_name</local_identifier>
     <nillable_flag>false</nillable_flag>
     <submitter_name>ACR</submitter_name>
     <definition>
-      The name of the detector that recorded the observation(s) comprising the
-      product. For instruments with multiple detectors, as in the case of MVIC, 
-      for example, this string will include both the instrument ID and the 
-      specific detector ID.
+      The nh:detector_name provides the identifier of the detector that recorded 
+      the observation(s) comprising the product. For instruments with multiple 
+      detectors, as in the case of MVIC, for example, this string will include 
+      both the instrument ID and the specific detector ID.
     </definition>
     <DD_Value_Domain>
       <enumeration_flag>true</enumeration_flag>
       <value_data_type>ASCII_Short_String_Collapsed</value_data_type>
+      <DD_Permissible_Value>
+        <value>Linear Etalon Imaging Spectral Array</value>
+        <value_meaning>
+          Data were obtained using the RALPH/LEISA infrared detector with a waveleght range of 1250 - 2500 nm. 
+        </value_meaning>
+      </DD_Permissible_Value>
+      <DD_Permissible_Value>
+        <value>Long-Range Reconnaissance Imager</value>
```

Comment:
I don't understand why this is a conflict. This appears to reference something in another development branch, not something included in this PR. This is what the biweekly (perhaps they should be weekly?) meetings are supposed to sort out, along with us reviewing each others PRs. Since that branch only appeared yesterday, there hasn't been a meeting yet in which to discuss it...

Reply:
It's a conflict in the sense that this PR contains LORRI content that is different than what was added here. If we try to merge both things into the same place, there will be a merge conflict and we will have to pick one or the other.

Reply:
Noted, and Ben has gotten his heads-up. The value will remain the same, but aspects of the definitions might be merged. That will be a separate action after this merge, though.